faults/description.go

```diff
@@ -36,6 +36,7 @@ type Description struct {
 	Parameters Parameters
 	OnFault    func(Description, Parameters) error
 	Count      int64
+	Error      string
```

Comment:
`Error` might be misleading as a member name here, what about `Description` or `FaultDescription`?